jekyll/_cci2/code-coverage.md

```diff
@@ -135,7 +220,80 @@ coverage html  # open htmlcov/index.html in a browser
 The generated files will be found under `htmlcov/`, which can be uploaded in a
 `store_artifacts` step in your config:
 
+{:.tab.python_example.Cloud}
+```yaml
+version: 2.1
+orbs:
+  browser-tools: circleci/browser-tools@1.2.3
+jobs:
+  build:
+    docker:
+    - image: cimg/python:3.10-browsers
+      auth:
+        username: mydockerhub-user
+        password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+    - checkout
+    - run:
```

Comment:
If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`